cli/gluster-block.c

```diff
@@ -566,6 +582,7 @@ glusterBlockCreate(int argcount, char **options, int json)
   blockCreateCli cobj = {{0}, };
   bool TAKE_SIZE=true;
   bool PREALLOC_OPT=false;
+  int i;
```

Comment:
Please fix:

gluster-block.c: In function 'glusterBlockCreate':
gluster-block.c:585:7: warning: unused variable 'i' [-Wunused-variable]
   int i;
       ^

Reply:
@lxbsz sure, thanks!